coordinator/ethereum/test-utils/src/test/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelperKtTest.kt

```diff
@@ -23,8 +23,8 @@ class MakefileContractDeploymentHelperKtTest {
     assertThat(
       getDeployedAddress(
         listOf(
-          "L2MessageServiceV1.2.3 artifact has been deployed in 1.2659626659999998s ",
-          "L2MessageServiceV1.2.3 deployed: address=0xFE48d65B84AA0E23594Fd585c11cAD831F90AcB6 blockNumber=8",
+          "contract=L2MessageServiceV1.2.3 artifact has been deployed in 1.2659626659999998s ",
```

Comment:
I think we haven't updated the TS code producing this log in `contracts/scripts/hardhat/utils.ts` - I see this pattern of log in `deployUpgradableFromFactory` and `deployUpgradableWithAbiAndByteCode` functions

So should we also update the console.log statements produced there? And if so, should we also update the other `... artifact has been deployed in ...` statements in this file?

Reply:
@kyzooghost - deployUpgradableFromFactory is supplementary and can potentially be revisited in another PR - the current changes are incorporated into the deploys that call the factories, so it would be a weird duplication.